stepper/stepper_controller/src/stepper_controller_node.py

```diff
@@ -0,0 +1,77 @@
+#! /usr/bin/env python
+
+# ROS packages.
+import rospy
+
+# Stepper generated messages.
+from stepper_controller.msg import Run
+from stepper_controller.msg import Move
+from stepper_controller.msg import GoTo
+from stepper_controller.msg import Stop
```

Comment:
Importing theses messages in node.js might be a pain. Maybe using std_msgs.String and using JSON would be a more standardised way of communicating?